examples/libh2o/latency-optimization.c

```diff
@@ -354,7 +354,12 @@ int main(int argc, char **argv)
 
     if (mode_listen) {
         int fd, reuseaddr_flag = 1;
-        if ((fd = socket(AF_INET, SOCK_STREAM, 0)) == -1 ||
+        if (
+#ifdef __linux__
+            (fd = socket(AF_INET, SOCK_STREAM | SOCK_CLOEXEC | SOCK_NONBLOCK, 0)) == -1 ||
+#else
+            (fd = socket(AF_INET, SOCK_STREAM, 0)) == -1 ||
+#endif
```

Comment:
I would appreciate it if you could revert the change to minimize the maintenance cost of the code.

Considering that a HTTP/2 connection is typically long-running and that latency optimization issues many syscalls than when set to off, I do not think we need to complicate things here.

Reply:
reverted.